src/test/java/com/connexta/store/adaptors/S3StorageAdaptorITests.java

```diff
@@ -153,11 +157,36 @@ public void testStoreWithContentLengthNotMatching() {
               10L,
               MediaType.APPLICATION_XML_VALUE,
               new ByteArrayInputStream(ASDF.getBytes()),
-              KEY,
+              DATASET_ID,
               Map.of());
         });
   }
 
+  @Test
+  void testUpdatingStoreStatus() {
```

Comment:
❗️I'd expect a test for updating a status for a datasetId that does not yet exist